src/client/stories/index.js

```diff
@@ -18,6 +24,14 @@ storiesOf("Button", module)
 const MapStories = storiesOf("Map", module).add("MapComponent", MapStory);
 MapStories.addDecorator(withKnobs);
 
+const VoyageStories = storiesOf("StartVoyage", module)
```

Comment:
Here "StartVoyage" is the label you will see in storybook. Please make it "Start Voyage" instead, so it is nice to read.

Reply:
Comment incorporated.